Ariane core test is still broken

The recent change definitely helped things, but the build is still not configured correctly. Most relevant slang error:
```
../../../third_party/cores/ariane/core/cache_subsystem/cva6_hpdcache_subsystem.sv:171:12: error: 'hpdcache_typedef.svh': No such file or directory
  `include "hpdcache_typedef.svh"
           ^~~~~~~~~~~~~~~~~~~~~~
```

--- a/generators/ariane
+++ b/generators/ariane
@@ -78,6 +78,9 @@ def process_filelist(filelist):
                 incdirs += l.partition("+incdir+")[2] + ' '
             elif l.startswith("+define+"):
                 defines += l.partition("+define+")[2] + ' '
+            elif l.startswith("-F"):
+                # recursively parse filelists
+                process_filelist(l.partition(' ')[2])
             elif l.endswith(".sv"):
                 sources.append(l)
 

